### ISSUE TITLE
"Case file" support

### ISSUE THREAD

**Issue by luis100** [keeps/db-visualization-toolkit#144]
User would like to be able to read through all documents in a case file instead of clicking through the registration of the document to view each document.

Sort documents according to case file number.

Also: Sort documents v in the case file according to date.

TBD: this issue needs more details

**Comment by Laurira** [comment]
I will ask Anders Bo or Phillip to get some good example for it.

**Comment by PhillipTommerholt** [comment]
Hi guys.
Here are my first thoughts on this. Let me know if it is not clear:

1) The users said they wanted "case file" support specifically, but the wish is rather this:

when you have searched and have got a proper result list of let's say 15 rows where there are LOBs in one column: then you wont have to make two clicks to open just one picture (one click to open the row data and one click to download the picture). You would like to have the opportunity to scroll through the 15 files. I have tried to make a Northwind SIARD 2.1 export with dbptk-app-2.6.1 and open it in DBVT-GUI 2.10 and push the "export all" and "export visible" buttons but there is a bug when I try to do that, so I am not sure whether you download the LOBs with this functionality? See my siardfile (I have renamed and put a .zip extension so that I can upload it here :
[northwind26new.siard.zip](https://github.com/keeps/db-visualization-toolkit/files/3783501/northwind26new.siard.zip) and the error:
![image](https://user-images.githubusercontent.com/15984159/67767241-80fe7c00-fa50-11e9-8ad6-ad5421117673.png)


2) Sort document according to case number

I guess this is a wish where the users want to combine searches across tables. If you have a table containing documents, but the sorting criteria exists in another table. How can one do that?
